app/upgrades/v10/upgrades_test.go

```diff
@@ -0,0 +1,61 @@
+package v10_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+
+	"github.com/osmosis-labs/osmosis/v9/app/apptesting"
+	v10 "github.com/osmosis-labs/osmosis/v9/app/upgrades/v10"
+)
+
+type UpgradeTestSuite struct {
+	apptesting.KeeperTestHelper
+}
+
+func (suite *UpgradeTestSuite) SetupTest() {
+	suite.Setup()
+}
+
+func TestKeeperTestSuite(t *testing.T) {
+	suite.Run(t, new(UpgradeTestSuite))
+}
+
+func (suite *UpgradeTestSuite) TestUpgradePayments() {
+	testCases := []struct {
+		msg         string
+		pre_update  func()
```

Comment:
Nit: Is there any reason we put the fields `pre_update` and `post_update`? If not could we delete these fields?

Reply:
good call, done